src/checkin/services/flow/step/GetExtraPassengerInformationStep.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { MockContext } from '@testMocks/model/Context.mock'
+import { Context } from '../../../model/Context'
+import GetExtraPassengerInformationStep from './GetExtraPassengerInformationStep'
+
+const MOCK_NAMES = ['Carlos', 'Juan', 'Emilio', 'Gary', 'Elias', 'Fernando', 'Giovanni']
+const MOCK_PASSPORTS = ['G143327', 'G113125', 'G229108', 'G229101']
+
+describe('[ Step / GetExtraPassengerInformationStep ]', () => {
+  const step = new GetExtraPassengerInformationStep()
+
+  describe('Test when it should be execute', () => {
+    it('should return true when passengers are more than one', () => {
+
+      const context = wireContextMockWithNumberOfPassengersInSessionData(2)
+
+      expect(step.when(context)).toBeTruthy()
+    })
+
+    it('should return false when there is only one passenger', () => {
+      const context = wireContextMockWithNumberOfPassengersInSessionData(1)
+
+      expect(step.when(context)).toBeFalsy()
+    })
+  })
+
+  describe('Test at execution', () => {
```

Comment:
In this test suite, the same value is created and assigned to the context for each test case:
`const context = wireContextMockWithNumberOfPassengersInSessionData(2).`
 Therefore, you might want to consider the possibility of using one of Jest's hooks: **beforeEach**. In this hook, you can assign the value to the context before each test, thus avoiding code duplication.

Reply:
Totally agree, thanks for the correction!